assets/training/model_management/environments/foundation-model-inference/context/requirements.txt

```diff
@@ -1,10 +1,10 @@
 deepspeed==0.12.5
 deepspeed-mii==0.1.3
-vllm==0.2.4
+vllm==0.3.2
 deepspeed-kernels==0.0.1.dev1698255861
```

Comment:
I see that we are using dev version of deepspeed-kernels. Any reason for this?

Reply:
I am not sure about context. Let me check with Miseon and share offline. As env release is urgent and we already have plenty versions with this dev version, will it be okay to push one more?

Reply:
Let's proceed and handle this in the next PR.